### ISSUE TITLE
Update 'peeling skin syndrome' (DOID:0060283) and add subtypes

### ISSUE THREAD

**Issue by allenbaron** [DiseaseOntology/HumanDiseaseOntology#1276]
### Is your request related to a specific disease? Please describe.
'peeling skin syndrome' (DOID:0060283) has been expanded by OMIM to a [phenotypic series](https://omim.org/phenotypicSeries/PS270300).

### Describe the proposed change(s)
- Add the phenotypic series subtypes
- Remove [OMIM:613088](https://omim.org/entry/613088) xref from DOID:0060283 --> this refers to PELVIC ORGAN PROLAPSE, SUSCEPTIBILITY TO, 2 which is not relevant to peeling skin (this may be a case where OMIM has re-used an ID)

**Comment by allenbaron** [comment]
Curation [sheet](https://docs.google.com/spreadsheets/d/1erVjra2bcCVn7ACtHekUZIQ2Ql8OV8sTVlZ4Je4GQhk/edit#gid=1794739170) with additional notes and robot template.

**Comment by allenbaron** [comment]
A couple of notes:
1. The subgroupings of 'peeling skin syndrome' (generalized type A/B, acral, etc.) are inconsistent so I have not included them. Where a subgroup corresponds to a single disease or is consistent it's been added as a broad synonym or mapping to the relevant term(s).
    - The confusion is especially true for the 2 diseases that Orphanet organized outside of the 'peeling skin syndrome' hierarchy and instead classified them as 'exfoliative ichthyosis'. The primary distinction seems to be the level at which the skin deficiency leading to peeling occurs. These have been described as basal/suprabasal but some reports report higher layers affected.
    - I've organized all of these as 'peeling skin syndrome'.
2. The causative variant of PSS3 may not be pathogenic (all other [ClinVar entries](https://www.ncbi.nlm.nih.gov/clinvar/variation/183645/?oq=RCV000162277+OR+RCV000890550&m=NM_001127895.2(CHST8):c.229C%3ET%20(p.Arg77Trp)) are likely benign or uncertain significance). I created the disease anyway but it may need to be deprecated if future reports call those results further into doubt. The relevant citation has been added with OMIM for their review.

More details are available in the curation sheet.
